src/styles/index.ts

```diff
@@ -1885,6 +1885,14 @@ const styles = (theme: ThemeColors) =>
             height: 40,
             padding: 10,
             margin: 3,
+            ...Platform.select({
```

Comment:
This is against our guidelines 😬

Reply:
> This is against our guidelines 😬

Oh I apologize. 

> what's the root cause?

Basically, I moved the `EmojiButton` and `SendButton` from `ReportActionCompose` to `ComposerWithSuggestions` and then it started to happen. The reason was to keep the re-rendering in one place only and don't re-render `ReportActionCompose`. It was basically concerning the empty state, which was previously defined in `ReportActionCompose`.

 Can you guys point me where can I look for design guidelines? OR anything relevant.

Reply:
Thanks, it seems quite complicated to move `SendButton` and `EmojiButton` to platform specific files and then apply the style there. I will see if I can find a way where we have to make less changes to the codebase,

Reply:
Alright, it's fixed now. I opted for a simpler approach and that's to take out `SendButton` and `EmojiButton` outside of the `View` encapsulating `Composer` in `ComposeWithSuggestions`.